src/main.ts

```diff
@@ -20,20 +20,22 @@ async function run() {
       {
         const environment = core.getInput('env', { required: true });
         const noOverride = core.getInput('no_override') === 'true';
-        let deploymentID = core.getInput('deployment_id', { required: false });
-        console.log(`initializing deployment ${deploymentID} for ${environment}`);
+        const transient = core.getInput('transient') === 'true';
+        const git_ref = core.getInput('ref') || ref;
```

Comment:
a nit for consistency
```suggestion
        const gitRef = core.getInput('ref') || ref;
```

```suggestion
        const gitRef = core.getInput('ref') || ref;
```

Reply:
🤦 I was switching back and forth between this and Ruby. Thanks for catching it.